components/Footer.js

```diff
@@ -0,0 +1,32 @@
+import HouseLine from "@/public/Icons/home-4-line.svg";
+import styled from "styled-components";
+import PlantFavIcon from "@/public/Icons/plant-line.svg";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <StyledLink href={"/"}>
+        <HouseLine />
+      </StyledLink>
+      <StyledLink href={"/myPlants"}>
+        <PlantFavIcon />
+      </StyledLink>
+    </StyledFooter>
+  );
+}
+
+const StyledFooter = styled.div`
```

Comment:
Since this is a footer, please use the `<footer>` HTML tag instead of a `<div>`. This semantic change will make the structure more accessible and improve SEO.
```suggestion
const StyledFooter = styled.footer
```

Reply:
Why didn't you change this?